redhat-developer/intellij-quarkus
# Ensure correct behavior with Spring applications

- Remove the Quarkus properties file type and use the standard Properties
- Add an IconProvider to decorate Quarkus application.properties

Fixes #51

Signed-off-by: Jeff MAURY <jmaury@redhat.com>

# How to test

1. Create a **SpringBoot** project and verify code assist works on `application.properties`
2. Create a **Quarkus** project and verify code assist works on `application.properties`
